scalahost/nsc/src/main/scala/scala/meta/internal/semantic/Inferred.scala

```diff
@@ -0,0 +1,36 @@
+package scala.meta.internal.semantic
+
+// data structure to manage multiple inferred sugars at the same position.
```

Comment:
That's an interesting thought, it would greatly simplify `Inferred.syntax` which is a win. The latest changes added a new `AttributedSugar` class which seems to be enough to handle conversions. However, what should the signature of `inferred: Map[Position, Inferred]` be then?